app/views/friends/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div id="find_friend_form">
+  <%= form_with url: '/friends', method: :post, local: true do |f| %>
+    <%= f.label :email, 'Email' %>
+    <%= f.text_field :email %>
+    <%= f.submit 'Find Friend' %>
+  <% end %>
+</div>
+
+<div id="search_results">
+  <% @users&.each do |user| %>
```

Comment:
Fancy here - I had to look this up. If anyone else is looking at this it will return nil if the item you're calling on it is nil rather than throwing an error.